player/command.c

```diff
@@ -2823,28 +2823,50 @@ static int mp_property_focused(void *ctx, struct m_property *prop,
     return m_property_bool_ro(action, arg, focused);
 }
 
+static int get_display_names_subkey(struct vo *vo, int action,
+                                    void *arg, enum mp_voctrl voctrl)
+{
+    switch (action) {
+        case M_PROPERTY_GET_TYPE:
```

Comment:
I'm gonna nitpick few style things. We have some some guidelines https://github.com/mpv-player/mpv/blob/master/DOCS/contribute.md , but general rule of thumb is to follow the stile of current file that is edited, in case there are differences.

cases should have the same indentation as switch